### PR TITLE
Express suggested changes before merging :)

### PR DESCRIPTION
**Pull request by dudustri**
- remove extra package jsons.
- fix env variable to use databasepath from the root

### CODE REVIEW

File: ExpressJS/app/src/app.js
Comment (dudustri) [inline ExpressJS/app/src/app.js:1]: I'm having issues to render the public page ... It shouldn't be my changes I think.

Comment (Ghmeding) [reply ExpressJS/app/src/app.js:1]: Fixed in other branch. We can delete this PR imo.

**Comment by dudustri** [comment]
Gustav chose to correct manually in his branch. Closing the PR

**Comment by dudustri** [comment]
Gustav chose to correct manually in his branch. Closing the PR

**Pull request closed without merging**